plugins/woocommerce-blocks/tests/e2e/tests/basic.block_theme.spec.ts

```diff
@@ -13,6 +13,12 @@ test.describe( 'Basic role-based functionality tests', () => {
 				page.getByRole( 'heading', { name: 'Dashboard' } )
 			).toHaveText( 'Dashboard' );
 		} );
+
+		test( 'Load post editor in iframe mode', async ( { page } ) => {
+			await page.goto( '/wp-admin/post-new.php' );
+
+			// Check that iframe with title "Editor canvas" is visible.
+			await expect( page.getByTitle( 'Editor canvas' ) ).toBeVisible();
```

Comment:
Wasn't sure what's the best way to go about this.

Reply:
My suggestion would be to use the [`frameLocator.owner`](https://playwright.dev/docs/api/class-framelocator#frame-locator-owner) API here:

```suggestion

		test( 'Load iframed post editor', async ( { admin, editor } ) => {
			await admin.createNewPost();
			await expect( editor.canvas.owner() ).toBeVisible();
```